safe/definitionsv4/report.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+
+"""
+Definitions for basic report
+"""
+from safe.reportv4.extractors.analysis_result import analysis_result_extractor
+from safe.reportv4.extractors.composer import qgis_composer_extractor
+from safe.reportv4.processors.default import qgis_composer_renderer, \
+    jinja2_renderer
+
+__author__ = 'Rizky Maulana Nugraha <lana.pcfre@gmail.com>'
```

Comment:
Please use the new header with these 4 lines : 
https://github.com/inasafe/inasafe/blob/develop/safe/impact_function_v4/impact_function.py#L63